components/ActionSheet/docs/typography-theming.md

```diff
@@ -1,39 +0,0 @@
-### Typography Theming
```

Comment:
Similarly here; please keep this around and annotate it as to-be-deleted.

Reply:
Done